### DISCUSSION TITLE
Error in Session Statistics and User Creation

### DISCUSSION THREAD

**Discussion by Muyu111** [DSJAS/DSJAS#66] [Installation help]
I have just completed the installation and it went well with the guidance of different posts and forums (thanks). However, some errors started to pop up here and there as I explored a little more.

When I try to check session statistics, I am prompted with:

Fatal error: Uncaught mysqli_sql_exception: Unknown column 'total_page_hits' in 'where clause' in C:\xampp\htdocs\include\DB.php:229 Stack trace: #0 C:\xampp\htdocs\include\DB.php(229): mysqli->query('SELECT `sys_dat...') #1 C:\xampp\htdocs\include\Stats.php(222): DB->unsafeQuery(Object(SimpleStatement)) #2 C:\xampp\htdocs\include\Stats.php(328): Statistics->isStatisticSystemReserved('total_page_hits') #3 C:\xampp\htdocs\admin\stats.php(135): Statistics->getStatistic('total_page_hits') #4 {main} thrown in C:\xampp\htdocs\include\DB.php on line 229

This message also appears in the transaction part of the fake bank and when I try to make a transfer with it.

When I try to change a user's password or delete a user I'm prompted with:

Warning: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\include\components\AdminSidebar.php:20) in C:\xampp\htdocs\admin\bank\editUser.php on line 100

Nonetheless, transactions and user creation continue to work, so I am confused as to what this problem is.

I have no experience with computers so hopefully, someone could give me a hand here.

Many thanks!

**Comment by ejv2** [comment]
Hi. As of commit 6c8ec3a, a few of these issues should be fixed (in particular the issues with header sequencing when editing users and creating transactions). Let me know if you have any more issues.
